include/zephyr/drivers/display.h

```diff
@@ -407,6 +426,42 @@ static inline int display_set_orientation(const struct device *dev,
 	return api->set_orientation(dev, orientation);
 }
 
+/**
+ * @brief Set display scroll area
+ *
+ * @param dev Pointer to device structure
+ * @param tfa Top Fixed Area (in No. of lines from Top of the Frame Memory and Display).
+ * @param bfa Bottom Fixed Area (in No. of lines from Bottom of the Frame Memory and Display).  
+ *
+ * @retval 0 on success else negative errno code.
+ */
+static inline int display_set_scroll_area(const struct device *dev,
+					uint16_t tfa, 
+					uint16_t bfa)
+{
+	struct display_driver_api *api =
+		(struct display_driver_api *)dev->api;
+
+	return api->set_scroll_area(dev, tfa, bfa);
```

Comment:
There is no need to implement [dummy functionality to displays](https://github.com/zephyrproject-rtos/zephyr/pull/56765/commits/ff5a49d20651522f90a65673723c7e6e6cc588cd).
```c
	if (api->set_scroll_area) {
		return api->set_scroll_area(dev, tfa, bfa);
	}

	return -ENOSYS;
```

Reply:
Got it, will remove.